src/vt_stability.jl

```diff
@@ -227,20 +224,21 @@ function vt_stability(
     nmol_test .*= p_init / (z_lg * RT * sum(nmol_test))
 
     D_lg = D_min(nmol_test, optmethod)
-    results[3] = VTStabilityResult{T}(D_lg ≥ thresh, D_lg, optmethod.x)
+    results[3] = VTStabilityResult{T}(!isnan(D_gl), D_lg ≥ thresh, D_lg, optmethod.x)
 
     # Test - liquid
     z_ll = compressibility(mix, nmol_test, p_init, RT, 'l')
     nmol_test .*= z_lg / z_ll
 
     D_ll = D_min(nmol_test, optmethod)
-    results[4] = VTStabilityResult{T}(D_ll ≥ thresh, D_ll, optmethod.x)
+    results[4] = VTStabilityResult{T}(!isnan(D_gl), D_ll ≥ thresh, D_ll, optmethod.x)
 
     if isnan(D_gg) && isnan(D_gl) && isnan(D_lg) && isnan(D_ll)  # все 4 попытки провалились
         error("VTStability: all tries have failed")
     end
 
-    isstable = results[1].isstable && results[2].isstable && results[3].isstable && results[4].isstable
+    converged = any(x -> x.converged, results)
```

Comment:
vt-stability is treated as "converged" when at least on of results is converged.